src/main/scala-2.12/org/apache/james/gatling/control/RecipientFeeder.scala

```diff
@@ -18,10 +19,11 @@ object RecipientFeeder {
           recipientSessionParam -> recipient.address))
       .toArray
 
-  def usersToFeeder(users: Seq[User]): RecipientFeeder =
+  def usersToFeeder(users: Seq[User]): RecipientFeederBuilder =
     users
       .map(user =>
         Map(
           recipientSessionParam -> user.username.value))
       .toArray
-}
+      .random
+  }
```

Comment:
is it an accidental indent or does it make sense? (hard to figure out in github)

Reply:
it is supposed to be aligned with `.toArray`

Reply:
I was asking for the closing curly brace